guides/Dataset.md

```diff
@@ -645,35 +641,58 @@ A point, or coordinate, would defined in this way:
 <pre>
   <strong>"box": "39.3280 120.1633 40.445 123.7878"</strong>
 </pre>
 
-The defined spatial coverages are inadequate for the needs of our community, but we also recognize that schema.org continues to hear the needs of its schema.org publishers on these [issues](https://github.com/schemaorg/schemaorg/issues/1548).
+NOTE-- see [discussion in GitHub issue 101](https://github.com/ESIPFed/science-on-schema.org/issues/101#issuecomment-720808142) on what works with Google Dataset search to display spatial locatation in their search results.
+
+East longitude values can be reported 0 <= X <= 360 or -180 <= X <= 180. Some applications will fail under one or the other of these conventions. Recommendation is to use -180 <= X <= 180, consistent with the [WKT specification](https://docs.opengeospatial.org/is/18-010r7/18-010r7.html#33).  Following this recommendation, bounding boxes that cross the antimeridian at ±180° longitude, the West longitude value will be numerically greater than the East longitude value. For example, to describe Fiji the box might be
+<pre>
+  <strong>"box": "-19 176 -15 -178"</strong>
+</pre>
+
+NOTES: Some spatial data processors will not correctly interpret the bounding coordinates across the antimeridian even if they follow the recommended southwest, northeast corner convention, resulting in boxes that span the circumference of the Earth, excluding the actual area of interest. For applications operating with data in the vicinity of longitude 180, testing is strongly recommended to determine if it works for bounding boxes crossing the antimeridian (+/- 180); an alternative is to define two bounding boxes, one on each side of 180.
 
-We also recognize that there is no defined property for specifying a Coordinate Reference System, but we see from the [schema.org issue queue](https://github.com/schemaorg/schemaorg/issues) that this has been mentioned.
+For bounding boxes that include the north or south pole, schema:box will not work. Recommended practice is to use a schema:polygon to describe spatial location extents that include the poles.  
 
-<a id="spatial_multiple-geometries"></a>
+<a id="spatial_multiple-geometries">Multiple geometries</a>
```

Comment:
Does it matter if order of geometries is not preserved? Perhaps a note like:

Note that ordering of array elements is not preserved in JSON-LD processors. If order is important a `@container` of `@list` should be specified. For example:
```
"geo": {
  "@list": [ 
    {"@type": "GeoCoordinates", "latitude": -17.56, "longitude":50},
    {"@type": "GeoCoordinates", "latitude": -19, "longitude":51}
  ]
}
```

Reply:
@smrgeoinfo for the record, at DataONE our harvester and indexer supports multiple discontiguous coverage values from metadata like EML. I'd have to check on ISO. We are not yet harvesting schema.org for that, but will keep it in mind when we do that indexing work.